weave-js/src/components/PagePanelComponents/Home/Browse3/filters/FilterBar.tsx

```diff
@@ -256,6 +309,20 @@ export const FilterBar = ({
 
   const onRemoveFilter = useCallback(
     (filterId: FilterId) => {
+      // Check if this is the first datetime filter
+      const sortedItems = sortDatetimeFirst([...localFilterModel.items]);
```

Comment:
What is the benefit of showing "Called" first in the dropdown?

Reply:
good point, removed.